lexic.txt

```diff
@@ -1,37 +0,0 @@
-Alphabet:
-
-a. Upper (A-Z) and lower case letters (a-z) of the English alphabet
-b. Underline character '_';
-c. Decimal digits (0-9);
-
--operators + - * / < <= = >= eq mod ** not
-
-- separators [ ] { } ; space
-
-- reserved words: init verify otherwise escape true untrue while for func in
-
-integer:
-
-<int>::= 0 | <nonzero>
-<nonzero>::=<nonzerodigit>|-<nonzerodigit>|<nonzerodigit><digitseq>|-<nonzerodigit><digitseq>
-<digitseq>::= <digit>|<digit><digitseq>
-<digit>::=0|<nonzerodigit>
-<nonzerodigit>::=1...
-
-
- <letter> ::= "a" | "b" | ... | "Z"
- <specialchar> ::= "_"
- <identifier> ::= <letter>|<ls><ldsequence>
-
- <ls>::=<letter>|<specialchar>
- <ldssequence> ::= <lds>|<lds><ldssequence>
- <lds>::=<letter>|<specialchar>|<digit>
- <char> ::= '<letter>|<digit>|<specialchar>'
- <string> ::= <char>|<string>
- <conststring>::="<string>"
- <const>::=<conststring>|<int>
-
- <func> ::= func <identifier>() | func <identifier>(<args>)
```

Comment:
You want to go pretty far if you want to add functions as well :D

Reply:
I will 100% regret this :))